app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ForgotPasswordsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:email])
+
+    if user
+      AppMailer.send_password_reinitialization_email(user).deliver
+      redirect_to forgot_password_confirmation_path 
+    else
+      flash[:error] = set_error_message(params[:email])
```

Comment:
You don't have to pass in the params, because params is available in the whole controller.